server/lib/resolvers/resolverMapper.js

```diff
@@ -42,8 +42,22 @@ module.exports = function (dataSources, resolverMappings, pubsub) {
       throw new Error(`Resolver builder for ${dataSource.type} missing buildResolver function`)
     }
 
-    const compiledRequestMapping = compile(resolverMapping.requestMapping)
-    const compiledResponseMapping = compile(resolverMapping.responseMapping)
+    const compile = (dataSource.type === 'Postgres') ? compileScript : compileTemplate
```

Comment:
However we can improve this code to make direct mapping between dataSource and method. For example
`
ResolverCompiler.compileFor(dataSource)
`